cmd/launcher/main.go

```diff
@@ -163,11 +163,11 @@ func runNewerLauncherIfAvailable(ctx context.Context, logger log.Logger) {
 	}
 
 	if newerBinary == "" {
-		level.Debug(logger).Log("msg", "nothing newer")
+		level.Info(logger).Log("msg", "nothing newer")
```

Comment:
Hrm. 

Worth noting that all Info logs land in the _system_ log files. So this is potentially noisy for customers that have centralized windows event logging.

Reply:
We can probably revert at least some of these back down to the debug level, now that I've worked through the issue I was trying to troubleshoot 🙂